### DISCUSSION TITLE
vlt package manager is not recognized by turborepo

### DISCUSSION THREAD

**Discussion by hamlim** [vercel/turborepo#9436] [Ideas]
### Verify canary release

- [X] I verified that the issue exists in the latest Turborepo canary release.

### Link to code that reproduces this issue

https://github.com/hamlim/vlt-monorepo

### What package manager are you using / does the bug impact?

~~npm~~
_vlt_

### What operating system are you using?

Mac

### Which canary version will you have in your reproduction?

2.2.3

### Describe the Bug

The new `vlt` package manager doesn't seem to be recognized yet.

### Expected Behavior

The `vlt` package manager should be recognized by turborepo

### To Reproduce

- Clone the reproduction repo: https://github.com/hamlim/vlt-monorepo
- Run `vlt install` (install `vlt` via something like `bun add -g vlt@latest` or `npm install -g vlt@latest`, etc)
- Run `vlt run dev` and see turborepo error

**Comment by hamlim** [comment]
Note - the issue template also only allows me to select a set list of package managers that doesn't list `vlt` yet either

**Comment by anthonyshew** [comment]
HI, we're not looking to support any new package managers right now. Maybe in the future! Will leave this open for posterity and anyone who wants to add their interest can add a 👍 to the top-level.

**Comment by hamlim** [comment]
Is there documentation on why there's a strict allowlist of package managers that turborepo works with as opposed to being agnostic of the package manager?

**Comment by anthonyshew** [comment]
We parse lockfiles to get knowledge about the package and task graph, letting us create all the optimizations that we do. Since every package manager has a different lockfile format, we have to write fresh code for every one.

**Comment by anthonyshew** [comment]
Converting to a Discussion as that's where we handle our feature requests.
